scripts/bump.sh

```diff
@@ -8,6 +8,7 @@ DEPENDENCIES=(
 
 # Variables
 NB_ARGS="$#"
+USAGE="bump.sh [-h] -p|-m|-M"
```

Comment:
as with `git`, i would find
```
bump.sh [-h | --help] [-p | --patch] [-m | --minor] [-M | --major]
```
quite nice and clear

Reply:
Actually it looks nice I agree but I wanted to make it clear that one of these arguments had to be given!

Reply:
> Actually it looks nice I agree but I wanted to make it clear that one of these arguments had to be given!
>
yup, that's an issue i have with the current usage i think...

> Have you something in mind to express that? If not no worries I'll just replace USAGE with your proposal!
>
we should find a usage that express "one and only one option has to be given" :thinking: 

i do not really know tbh :eyes:

Reply:
> we should find a usage that express "one and only one option has to be given" thinking

Actually `bump.sh -p|-m|-M` expresses that, I think?

Reply:
but you changed to my proposition right? :thinking:

Reply:
ok saw that in your comment below :+1: